pkg/backup/client.go

```diff
@@ -571,10 +571,10 @@ func (bc *Client) findRegionLeader(ctx context.Context, key []byte) (*metapb.Pee
 		}
 		if region.Leader != nil {
 			log.Info("find leader",
-				zap.Reflect("Leader", region.Leader), zap.Stringer("Key", logutil.WrapKey(key)))
+				zap.Reflect("Leader", region.Leader), logutil.Redact(zap.Stringer("Key", logutil.WrapKey(key))))
 			return region.Leader, nil
 		}
-		log.Warn("no region found", zap.Stringer("Key", logutil.WrapKey(key)))
+		log.Warn("no region found", logutil.Redact(zap.Stringer("Key", logutil.WrapKey(key))))
```

Comment:
What about redacting keys in `logutil.WrapKey`?

Reply:
How does TiDB/PD redact key and values?

Reply:
https://github.com/pingcap/tidb/pull/21141/files
https://github.com/pingcap/tidb/pull/19409/files
TiDB uses `if else` to redact SQLs. They can get the configuration from the session.

Our usage is similar to PD's. https://github.com/tikv/pd/blob/master/pkg/logutil/log.go#L302